algorithms/1-intro/intro_algo_answers.txt

```diff
@@ -0,0 +1,56 @@
+1.
+  a.)
+  DEF MAKE_AN_OMELETTE
+    HEAT pan on stove
+    CRACK eggs into mixing bowl
+    ADD milk TO bowl
+    REPEAT
+      whisk eggs
+    UNTIL eggs bubbly
+    ADD butter TO pan
+    IF butter completely melted
+      ADD mixed eggs to pan
+      REPEAT
+        tilt and turn pan
+      UNTIL pan covered entirely
+    END IF
+    IF mixed eggs not runny
+      ADD filling
+      FOLD egg
+    END IF
+    SET COUNTER to 0
+    WHILE COUNTER less than 31
+      COUNTER + 1
+    END WHILE
+    REMOVE folded egg mixture
+    EAT
+  END
+
+  b.)
+  DEF TIP(BILL, QUALITY OF SERVICE)
+    IF QUALITY OF SERVICE is normal
+      RETURN BILL * 0.2
```

Comment:
I would avoid doing the returns here. Instead, I would have this procedure explicitly hand back the bill. This helps accomplish the comment below.